src/hotspot/share/opto/block.cpp

```diff
@@ -603,6 +603,19 @@ void PhaseCFG::convert_NeverBranch_to_Goto(Block *b) {
   dead->head()->del_req(j);
   for( int k = 1; dead->get_node(k)->is_Phi(); k++ )
     dead->get_node(k)->del_req(j);
+  // If the fake exit block becomes unreachable, remove it from the block list.
```

Comment:
Can you add missing {} in preceding code (lines 593 - 605)?

Reply:
Done.